ui/src/components/Drawer/index.vue

```diff
@@ -47,7 +47,7 @@ const drawerTabs = [
 
 const hasToken = ref(false);
 const showEmpty = ref(false);
-const drawerStatus = ref(true);
+const drawerStatus = ref(false);
 const isRequestingToken = ref(false);
 const fileManagerToken = ref('');
 const timeoutId = ref<number | null>(null);
```

Comment:
It appears there might have been a typographical error in line 9 of the code snippet you've shared. The function name seems to be missing after "fileManagerToken", which looks different from its counterpart inside the `draw` function in lines -33 -36 and -41. You can correct this issue by ensuring that it aligns perfectly before making further changes or adjustments:

```
@@-68,7 +68,7 @@ 
 const getAuthTokenAsync(): Observable<{ status: 'success' | 'fail', result: { token: string } }> {
   return http.request<any>({
     method: HTTP.Method.GET,
     url: `${process.env.NEXT_PUBLIC_API}/getToken`,
       ```diff

```